plugins/woocommerce-blocks/assets/js/blocks/attribute-filter/block.tsx

```diff
@@ -519,6 +519,21 @@ const AttributeFilterBlock = ( {
 		heading
 	);
 
+	const descriptionId = `attribute-filter-description-${
+		blockAttributes.attributeId
+	}-${ Math.floor( Math.random() * 100 ) }-${ Date.now() }`;
```

Comment:
@dkotter Since the page can have multiple instances of this block, a unique ID is required to ensure each block has its own description.

Unfortunately, I didn't find any JS function in wp packages similar to the `wp_unique_id` PHP function.

That's why I used a combination of `Math.random` with `Date.now` methods.
Do you see a problem with this approach?

Reply:
I think this is probably fine, though I believe the `uuid` package is used in a few places in Woo so that may be something we can access here

Reply:
The `package.json` of this block doesn't include the `uuid` library. To avoid installing it I went for that approach.
Thanks!